docs/source/tutorial/builtin_components.rst

```diff
@@ -63,34 +63,49 @@ Configuration
    Open your Django project's ``settings.py`` and add the following applications to ``INSTALLED_APPS`` -
    you only need (and should) add the components you want content editors to use:
 
-   .. code-block:: python
-
-      INSTALLED_APPS = [
-          # Optional dependencies
-          'djangocms_icon',
-          'easy_thumbnails',
-          'djangocms_link',  # Required if djangocms_frontend.contrib.link is used
-          # Main frontend components
-          'djangocms_frontend',
-          'djangocms_frontend.contrib.accordion',
-          'djangocms_frontend.contrib.alert',
-          'djangocms_frontend.contrib.badge',
-          'djangocms_frontend.contrib.card',
-          'djangocms_frontend.contrib.carousel',
-          'djangocms_frontend.contrib.collapse',
-          'djangocms_frontend.contrib.component',
-          'djangocms_frontend.contrib.content',
-          'djangocms_frontend.contrib.grid',
-          'djangocms_frontend.contrib.icon',
-          'djangocms_frontend.contrib.image',
-          'djangocms_frontend.contrib.jumbotron',
-          'djangocms_frontend.contrib.link',
-          'djangocms_frontend.contrib.listgroup',
-          'djangocms_frontend.contrib.media',
-          'djangocms_frontend.contrib.tabs',
-          'djangocms_frontend.contrib.utilities',
+    .. code-block:: python
+
+        INSTALLED_APPS = [
+            # Optional dependencies
+            'djangocms_icon',
+            'easy_thumbnails',
+            'djangocms_link',  # Required if djangocms_frontend.contrib.link is used
+            # Main frontend components
+            'djangocms_frontend',
+            'djangocms_frontend.contrib.accordion',
+            'djangocms_frontend.contrib.alert',
+            'djangocms_frontend.contrib.badge',
+            'djangocms_frontend.contrib.card',
+            'djangocms_frontend.contrib.carousel',
+            'djangocms_frontend.contrib.collapse',
+            'djangocms_frontend.contrib.component',
+            'djangocms_frontend.contrib.content',
+            'djangocms_frontend.contrib.grid',
+            'djangocms_frontend.contrib.icon',
+            'djangocms_frontend.contrib.image',
+            'djangocms_frontend.contrib.jumbotron',
+            'djangocms_frontend.contrib.link',
+            'djangocms_frontend.contrib.listgroup',
+            'djangocms_frontend.contrib.media',
+            'djangocms_frontend.contrib.tabs',
+            'djangocms_frontend.contrib.utilities',
       ]
 
+    For example, if you don't want to use any built-in components because you plan on 
+    :ref:`building your own <custom_components>`, a minimal setup of ``INSTALLED_APPS`` 
+    would look like this:
+
+    .. code-block:: python
+
+        INSTALLED_APPS = [
+            'easy_thumbnails',
+            'djangocms_link',  # Required if djangocms_frontend.contrib.link is used
+            # Main frontend app - pre-built components not needed
+            'djangocms_frontend',
+        ]
```

Comment:
This part was brought from the other tutorials, I think it makes more sense here